src/components/Icon/Icon.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import paths from './Icons';
+
+
+
+/**
+ * An svg icon whose path may be specified with the 'kind' prop.
+ */
+const Icon = ({ kind, width, height, fill, className }) => {
+
+  const iconClass = classes(
+    'Icon',
+    className,
+    fill
+  );
+
+  return (
+    <svg
+      className={iconClass}
+      width={width}
+      height={height}
+      viewBox="0 0 24 24"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <defs />
+      <title>
+        icon-
+        {kind}
+      </title>
+      <g id={`${kind}-icon`} data-name={`${kind} icon`}>
+        <path
+          d={paths[kind]}
+          id="path"
```

Comment:
add `width` and `height` set to `100%` this will scale the path with the parent svg size